### PR TITLE
French translation mistakes fixed

### PR DESCRIPTION
**Pull request by WilliamAboucaya**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
Corrected wording, grammar and punctuation mistakes in the french version of the "starting-a-project" article. Fixes include :
* Wrong translations leading to misinterpretation like: **un projet d'ouverture d'entreprise** &rarr; **une entreprise qui ouvre le code source d'un projet**
* Wrong choice of words like: **Tout le monde essaie la tarte** &rarr; **Tout le monde goûte la tarte**
* Unusual punctuation. For example, in French, a "?" must be preceded by a whitespace (See differences in the [line 145 of the starting-a-project.md file](https://github.com/github/opensource.guide/compare/master...WilliamAboucaya:patch-1#diff-c79d8cd948ba12cd45d47e8c502f0451R145)).

### CODE REVIEW

File: _articles/fr/starting-a-project.md [hunk 1]
```diff
@@ -103,71 +103,71 @@ Si vous avez besoin d'un budget ou d'un personnel dédié pour la promotion, les
...
 Les fichiers README font plus qu'expliquer comment utiliser votre projet. Ils expliquent également pourquoi votre projet est important et ce que vos utilisateurs peuvent en faire.
 
-Dans votre fichier README, essayez de répondre aux questions suivantes:
+Dans votre fichier README, essayez de répondre aux questions suivantes :
 
-* Que fait ce projet?
-* Pourquoi ce projet est-il utile?
-* Comment puis-je commencer?
-* Où puis-je obtenir plus d'aide, si j'en ai besoin?
+* Que fait ce projet ?
```
Comment (MikeMcQuaid) [inline _articles/fr/starting-a-project.md]: Please avoid adding extra space here and elsewhere.

File: _articles/fr/starting-a-project.md [hunk 2]
```diff
@@ -22,12 +22,12 @@ L'open source est puissant car il abaisse les barrières à l'adoption, permetta
 
 Pour comprendre comment cela fonctionne, imaginez que votre ami a un potluck, et que vous apportez une tarte aux cerises.
 
-* Tout le monde essaie la tarte (_utiliser_)
-* La tarte est un succès! Ils vous demandent la recette que vous fournissez (_voir_)
-* Un ami, Alex, qui est un chef pâtissier, suggère de réduire le sucre (_modifier_)
+* Tout le monde goûte la tarte (_utiliser_)
+* La tarte est un succès ! Ils vous demandent la recette que vous leur fournissez (_voir_)
```
Comment (MikeMcQuaid) [inline _articles/fr/starting-a-project.md]: Why the extra space before `!`?

File: _articles/fr/starting-a-project.md (hunk 1 above)
Comment (WilliamAboucaya) [reply _articles/fr/starting-a-project.md]: Using the French punctuation rules, you should put a space before exclamation marks, question marks, colons, etc.
[Here](https://www.iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation)'s a quick guide explaining this rule of French grammar.

File: _articles/fr/starting-a-project.md (hunk 2 above)
Comment (WilliamAboucaya) [reply _articles/fr/starting-a-project.md]: *Idem*

**Pull request merged**